site/docs/v1/tech/apis/_tenant-response-body-base.adoc

```diff
@@ -552,6 +558,56 @@ This strategy instructions FusionAuth when to append a unique suffix to the user
+Whether the WebAuthn bootstrap workflow is enabled.
+
+[field]#{base_field_name}.webAuthnConfiguration.bootstrapWorkflow.userVerificationRequirement# [type]#[String]# [since]#Available since 1.41.0#::
+The user verification requirement for WebAuthn passkey registration and authentication when using the bootstrap workflow. The possible values are:
++
+* `discouraged` - If possible, do not require user verification during registration and authentication.
+* `preferred` - Prefer authenticators that support user verification during registration and authentication but allow others.
+* `required` - Only authenticators that support user verification will be allowed.
+
+[field]#{base_field_name}.webAuthnConfiguration.debug# [type]#[Boolean]# [since]#Available since 1.41.0#::
+Whether debug event log output is enabled for WebAuthn.
+
+[field]#{base_field_name}.webAuthnConfiguration.enabled# [type]#[Boolean]# [since]#Available since 1.41.0#::
+Whether WebAuthn configuration is enabled.
+
+[field]#{base_field_name}.webAuthnConfiguration.reauthenticationWorkflow.authenticatorAttachmentPreference# [type]#[String]# [since]#Available since 1.41.0#::
+The authenticator attachment requirement for WebAuthn passkey registration when using the reauthentication workflow. The possible values are:
++
+* `any` - An authenticator with any attachment modality will be allowed during registration.
+* `crossPlatform` - Only authenticators with the `cross-platform` attachment modality will be allowed during registration. These are also referred to as "roaming" authenticators.
+* `platform` - Only authenticators with the `platform` attachment modality will be allowed during registration.
+
+[field]#{base_field_name}.webAuthnConfiguration.reauthenticationWorkflow.enabled# [type]#[Boolean]# [since]#Available since 1.41.0#::
+Whether the WebAuthn reauthentication workflow is enabled.
+
+[field]#{base_field_name}.webAuthnConfiguration.reauthenticationWorkflow.userVerificationRequirement# [type]#[String]# [since]#Available since 1.41.0#::
+The user verification requirement for WebAuthn passkey registration and authentication when using the reauthentication workflow. The possible values are:
++
+* `discouraged` - If possible, do not require user verification during registration and authentication.
+* `preferred` - Prefer authenticators that support user verification during registration and authentication but allow others.
+* `required` - Only authenticators that support user verification will be allowed.
+
+[field]#{base_field_name}.webAuthnConfiguration.relyingPartyId# [type]#[String]# [since]#Available since 1.41.0#::
+The Relying Party Id the tenant will use in WebAuthn ceremonies. Passkeys can only be used to authenticate on sites with the same Relying Party Id they were registered.
++
+The default value is `null`, which behaves the same as using the fully qualified domain name.
```

Comment:
Q: so if someone sets the `null`, or leaves the default, `null` works for all other `nulls`?

That seems wrong when I write/think about it, but this is how I read it.

Reply:
If they don't provide a value, we set the value on the configuration to `null`. When `null` is used in the WebAuthn creation/request options passed to the client-side WebAuthn API, it uses the current fully qualified domain.

The passkeys are scoped the same whether they leave the configuration blank or set the configuration to the FQDN.

Does that help clear it up? Any recommendations to make the doc clearer?

Reply:
I reworded this because we don't default things to `null`. Let me know if that still doesn't make sense.